pySTEL/libstell/fusion.py

```diff
@@ -43,6 +42,59 @@ def __init__(self):
 		self.E_DD_He3   = 0.82E6*EC
 		self.E_DHe3_He4 = 3.60E6*EC
 		self.E_DHe3_p   = 1.47E7*EC
+		self.n_prof     = lambda rho,f0,fe,alpha: (f0-fe)*(1.0-rho**alpha)+fe
+		self.t_prof     = lambda rho,f0,fe,alpha: (f0-fe)*(1.0-rho**alpha)+fe
+
+	def iss04(self,a,R,P,navg,B0,iota):
+		"""Computes the ISS04 scaling
+
+		This routine computes the ISS04 scaling as defined in:
+		https://doi.org/10.1088/0029-5515/45/12/024
+
+		Parameters
+		----------
+		a : real
+			Average minor radius [m]
+		R : real
+			Average major radius [m]
+		P : real
+			Total Power [MW]
+		navg : real
+			Average electron density 1E19 [m^-3]
+		B0 : real
+			Average magnetic field [T]
+		iota : real
+			Rotational transform at 2/3 flux [arb]
+		Returns
+		----------
+		tauiss04 : real
+			Confinement time [s]
+		"""
+		return 0.134*a**2.28*R**0.64*P**-0.64*navg**0.54*B0**0.84*iota**0.41
```

Comment:
Exponent of Power is -0.61.

Reply:
Corrected these two issues in the recent push.